pom.xml

```diff
@@ -310,6 +310,9 @@
         <plugin>
           <artifactId>maven-jar-plugin</artifactId>
           <version>2.6</version>
+          <configuration>
+            <excludes>the.hpl,InjectedTest.class,test-dependencies/</excludes>
```

Comment:
`the.hpl` and `InjectedTest` included local file paths, and are not used by other plugins depending on this one. But `test-dependencies` was the main problem.